src/main/java/seedu/duke/TaskList.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * Project name: Duke
+ * @author Tan Jun Da A023489eU
+ */
+
+package seedu.duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.duke.task.Deadline;
+import seedu.duke.task.Event;
+import seedu.duke.task.Task;
+import seedu.duke.task.Todo;
+
+/**
+ * Represents a taskList in an ArrayList format.
+ * A <code>TaskList</code> object corresponds to
+ * an ArrayList.
+ */
+public class TaskList {
+
+    protected List<Task> tasksList;
+    protected int counter;
```

Comment:
Suggestion: give this a more specific name, ie taskCounter.

Reply:
Ok, i will make these changes. 👍